std_workflow_dev/js/std_workflow_debug.js

```diff
@@ -0,0 +1,131 @@
+/* Haplo Platform                                     http://haplo.org
+ * (c) Haplo Services Ltd 2006 - 2016    http://www.haplo-services.com
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.         */
+
+P.onLoad = function() {
+    if(!O.PLUGIN_DEBUGGING_ENABLED) {
+        throw new Error("DEV PLUGIN std_workflow_dev LOADED ON NON-DEV SERVER");
+    }
+};
+
+if(O.PLUGIN_DEBUGGING_ENABLED) {
+
+    P.workflow.registerOnLoadCallback(function(workflows) {
+        var showDebugTools = function() {
+            return (O.PLUGIN_DEBUGGING_ENABLED &&
+                O.currentAuthenticatedUser &&
+                O.currentAuthenticatedUser.isSuperUser &&
```

Comment:
This stuff is actually probably useful for support. Maybe these things do belong in the standard plugin?

Reply:
Do you mean showing of admin tools while impersonating? Main worry for me is confusing users but maybe one for further down the line?